tests/test_metadata.py

```diff
@@ -44,6 +45,31 @@ def test_pass(metadata):
     assert result.ret == 0
 
 
+def test_additional_metadata_from_json(testdir):
+    testdir.makepyfile(
+        """
+        def test_pass(metadata):
+            assert metadata.get('Imran') == 'Mumtaz'
+    """
+    )
+    result = testdir.runpytest("--metadata-from-json", '{"Imran": "Mumtaz"}')
+    assert result.ret == 0
+
+
+def test_additional_metadata_using_both_key_values_and_json(testdir):
+    testdir.makepyfile(
+        """
+        def test_pass(metadata):
+            assert metadata.get('John') == 'Cena'
+            assert metadata.get('Dwayne') == 'Johnson'
```

Comment:
WWE fan? 😊

Reply:
Haha I named my cats after them. So I guess more of a fan of my cats 😆 .